Makefile

```diff
@@ -115,13 +115,22 @@ versioninfo:
 	go run scripts/generate_versioninfo/main.go; \
 	go generate cmd/telegraf/telegraf_windows.go; \
 
+.PHONY: build_generator
+build_generator:
+	go build -o ./tools/readme_config_includer/generator ./tools/readme_config_includer/generator.go
+
+insert_config_to_readme_%: build_generator
+	go generate -run="readme_config_includer/generator$$" ./plugins/$*/...
+
+generate_plugins_%: build_generator
+	go generate -run="plugindata/main.go$$" ./plugins/$*/...
+
 .PHONY: generate
-generate:
-	go generate -run="plugindata/main.go$$" ./plugins/inputs/... ./plugins/outputs/... ./plugins/processors/... ./plugins/aggregators/...
+generate: insert_config_to_readme_inputs generate_plugins_outputs generate_plugins_processors generate_plugins_aggregators
 
 .PHONY: generate-clean
 generate-clean:
-	go generate -run="plugindata/main.go --clean" ./plugins/inputs/... ./plugins/outputs/... ./plugins/processors/... ./plugins/aggregators/...
+	go generate -run="plugindata/main.go --clean" ./plugins/outputs/... ./plugins/processors/... ./plugins/aggregators/...
```

Comment:
Is the `generate-clean command` still used anywhere? Should it be deleted?

Reply:
Currently it is useful for `outputs`, `processors` and `aggregators`. I will remove it when everything is migrated.